dual_net.py

```diff
@@ -470,71 +464,31 @@ def bootstrap(working_dir):
         tf.train.Saver().save(sess, save_file)
 
 
-def export_model(working_dir, model_path):
+def export_model(model_path):
     """Take the latest checkpoint and export it to model_path for selfplay.
 
     Assumes that all relevant model files are prefixed by the same name.
     (For example, foo.index, foo.meta and foo.data-00000-of-00001).
 
     Args:
-        working_dir: The directory where tf.estimator keeps its checkpoints
         model_path: The path (can be a gs:// path) to export model to
     """
-    estimator = tf.estimator.Estimator(model_fn, model_dir=working_dir)
+    estimator = tf.estimator.Estimator(model_fn, model_dir=FLAGS.model_dir)
     latest_checkpoint = estimator.latest_checkpoint()
     all_checkpoint_files = tf.gfile.Glob(latest_checkpoint + '*')
     for filename in all_checkpoint_files:
         suffix = filename.partition(latest_checkpoint)[2]
         destination_path = model_path + suffix
         print("Copying {} to {}".format(filename, destination_path))
         tf.gfile.Copy(filename, destination_path)
+    # also export a .pb for C++ inference
+    n = DualNetwork(latest_checkpoint)
+    out_graph = tf.graph_util.convert_variables_to_constants(
```

Comment:
Have you tested this with `use_tpu` set to true?  The previous version of freeze did not run on TPUs.

Reply:
blargh -.-

I combined the calls because all usage locations of dual_net.export_model were accompanied by main.freeze_graph

I think what you're saying implies that we should have a guard at top saying `if FLAGS.use_tpu: raise Exception("exporting doesn't work on a tpu machine")` or something along those lines.